### ISSUE TITLE
Feature Request - Room Detection

### ISSUE THREAD

**Issue by R4ZZB3RRY** [fabianosan/HomeAssistantAssistAWS#3]
Hi!  There is another Alexa hosted conversation api integration on github and it has a room detection feature if your using AI.  Its a really great feature and I was wondering if you could add this to your code?  I'm not sure how to add this to the code myself, but all you need to do is to be able to send the device id of the alexa device your using with the query, and tell your AI to determine the area your in based of that.  

Let us know!

Thanks,

**Comment by fabianosan** [comment]
You already tested it?

I developed the hosted version and removed it from this version, because it not work all the time, because the IA have different behavior with same command form different areas, so, I disabled it. It make the responde a bit slow and it's complex to configure (not all users can do it in Home Assistant, taking the device ID from Alexa devices in they home)

But, if you test the other version (hosted) and confirm that is working fine, I can do the merge to this version, no problem.
